### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by jahnotto** [microsoft/MixedReality-WebRTC#87]
Hi,

I am getting a build error after pulling the lastest master (59df5a141edf80639b08e40480b7a2bca85595d8 [59df5a1]).

I am using Visual Studio 2019. I am not upgrading to a newer toolset or windows sdk when I'm loading the solution.

Any ideas?

```
2>Microsoft.MixedReality.WebRTC.Native.UWP.vcxproj -> C:\git-3rdparty\mr-webrtc\libs\Microsoft.MixedReality.WebRTC.Native\src\uwp\..\..\..\..\bin\UWP\ARM\Debug\Microsoft.MixedReality.WebRTC.Native.dll
2>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VisualStudio\v16.0\AppxPackage\Microsoft.AppXPackage.Targets(5991,5): error APPX1101: Payload contains two or more files with the same destination path 'Org.WebRtc.dll'. Source files: 
2>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VisualStudio\v16.0\AppxPackage\Microsoft.AppXPackage.Targets(5991,5): error APPX1101: C:\git-3rdparty\mr-webrtc\packages\Microsoft.MixedReality.WebRTC.Native.Core.WinRT.x86.Release.0.2.1-preview-20190917.1\build\native\..\..\runtimes\win10-x86\native\Org.WebRtc.dll
2>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VisualStudio\v16.0\AppxPackage\Microsoft.AppXPackage.Targets(5991,5): error APPX1101: C:\git-3rdparty\mr-webrtc\packages\Microsoft.MixedReality.WebRTC.Native.Core.WinRT.x86.Debug.0.2.1-preview-20190917.1\build\native\..\..\runtimes\win10-x86\native\Org.WebRtc.dll
2>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VisualStudio\v16.0\AppxPackage\Microsoft.AppXPackage.Targets(5991,5): error APPX1101: C:\git-3rdparty\mr-webrtc\packages\Microsoft.MixedReality.WebRTC.Native.Core.WinRT.x64.Release.0.2.1-preview-20190917.1\build\native\..\..\runtimes\win10-x64\native\Org.WebRtc.dll
2>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VisualStudio\v16.0\AppxPackage\Microsoft.AppXPackage.Targets(5991,5): error APPX1101: C:\git-3rdparty\mr-webrtc\packages\Microsoft.MixedReality.WebRTC.Native.Core.WinRT.x64.Debug.0.2.1-preview-20190917.1\build\native\..\..\runtimes\win10-x64\native\Org.WebRtc.dll
2>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VisualStudio\v16.0\AppxPackage\Microsoft.AppXPackage.Targets(5991,5): error APPX1101: C:\git-3rdparty\mr-webrtc\packages\Microsoft.MixedReality.WebRTC.Native.Core.WinRT.ARM.Release.0.2.1-preview-20190917.1\build\native\..\..\runtimes\win10-arm\native\Debug\Org.WebRtc.dll
2>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VisualStudio\v16.0\AppxPackage\Microsoft.AppXPackage.Targets(5991,5): error APPX1101: C:\git-3rdparty\mr-webrtc\packages\Microsoft.MixedReality.WebRTC.Native.Core.WinRT.ARM.Debug.0.2.1-preview-20190917.1\build\native\..\..\runtimes\win10-arm\native\Debug\Org.WebRtc.dll
2>Done building project "Microsoft.MixedReality.WebRTC.Native.UWP.vcxproj" -- FAILED.
```

**Comment by eirikhollis** [comment]
Same issue here, seems the DLLs are still generated in `C:\mr-webrtc\bin` though, so you could check that folder and see if they are there.

**Comment by jahnotto** [comment]
> Same issue here, seems the DLLs are still generated in `C:\mr-webrtc\bin` though, so you could check that folder and see if they are there.

They are there! Thanks.

**Comment by djee-ms** [comment]
I have experienced the same yesterday. This seems to be a regression, but I don't know yet if it's related to changes in the packages or to the latest Visual Studio update which brings support for the newly released .NET Core 3.0, because I had quite a lot of troubles with that and NuGet.

**Comment by djee-ms** [comment]
Actually I can confirm that on Visual Studio 16.1.2 all projects of the solution build correctly on 59df5a1 (latest `master`).

**Comment by eirikhollis** [comment]
Makes sense, I'm running 16.3.2 right now

**Comment by djee-ms** [comment]
And I can confirm on 16.3.2 it doesn't work anymore. This is an expected regression, because I was hoping to release a v1.0 at the end of the week, so that's why all regression bugs are popping up this week 😒

**Comment by djee-ms** [comment]
By the way you can temporarily work around the issue by commenting all but one `Microsoft.MixedReality.WebRTC.Native.Core.WinRT.*` package in the `packages.json` so that there's no conflict anymore. Pick the one you use most (_e.g._ x64 Debug).

**Comment by jahnotto** [comment]
Thanks for following up so quickly @djee-ms

**Comment by djee-ms** [comment]
https://github.com/microsoft/dotnet/issues/1116

**Comment by djee-ms** [comment]
Auto-closed by mistake, but should be fixed now with new packaged (0.2.2-preview-20191008.1) as of 8141ded471a169e0267c66f3d366a1d369bf6907.
